python_modules/dagster-airflow/dagster_airflow/scaffold.py

```diff
@@ -0,0 +1,164 @@
+# What we want to do here is consume a pipeline definition and a config, and then generate an
+# Airflow DAG definition, each of whose nodes corresponds to one step of the execution plan.
+
+import os
+
+try:
+    from black import format_file_contents
```

Comment:
I don't think we should be opinionated about code formatting. We should let teams use the formatted they want or no formatter at all